src/test/java/com/google/javascript/cl2dts/static_provided.d.ts

```diff
@@ -0,0 +1,27 @@
+declare namespace ಠ_ಠ.cl2dts_internal.goog.events.BrowserEvent {
```

Comment:
Same comment as in other CL: in the interest of future understandability, please consider making the names/types explain what the tested use case it, not just C&P from Closure library.